internal/character/kafka/talent.go

```diff
@@ -0,0 +1,98 @@
+package kafka
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	kafkaTalent = "kafka-talent"
+	followup    = "kafka-followup"
+)
+
+func init() {
+	modifier.Register(kafkaTalent, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnPhase2: restoreTalent,
+		},
+	})
+
+}
+
+func restoreTalent(mod *modifier.Instance) {
+	kafka, _ := mod.Engine().CharacterInstance(mod.Owner())
+	c := kafka.(*char)
+	c.canUseTalent = true
+}
+
+func (c *char) initTalent() {
+	c.engine.AddModifier(c.id, info.Modifier{
+		Name:   kafkaTalent,
+		Source: c.id,
+	})
+	c.engine.Events().AttackEnd.Subscribe(c.talentTrigger)
+	c.engine.Events().ActionEnd.Subscribe(c.talentAttack)
```

Comment:
There's a bunch of linter errors that suggest these funcs (and funcs/vars they call) aren't being used. I'm not sure why, as this seems fine at first glance. Something to investigate, I suppose.

Reply:
On further review, it seems that these aren't being used b/c of the previous inits not being called. So if the other comments are fixed, these should be too.